dashboard/test/helpers/plc_evaluation_helper_test.rb

```diff
@@ -0,0 +1,120 @@
+require 'test_helper'
+
+class PlcEvaluationHelperTest < ActionView::TestCase
+  include Plc::EvaluationsHelper
+
+  setup do
+    course = create :plc_course
+    @course_unit = create(:plc_course_unit, plc_course: course)
+
+    @stage_cliffs = create(:stage, name: 'Cliff stage', script: @course_unit.script, flex_category: Plc::LearningModule::CONTENT_MODULE)
+    @stage_ornithology = create(:stage, name: 'Ornithology stage', script: @course_unit.script, flex_category: Plc::LearningModule::CONTENT_MODULE)
+    @stage_ignorance = create(:stage, name: 'Ignorance stage', script: @course_unit.script, flex_category: Plc::LearningModule::CONTENT_MODULE)
+    @stage_blue = create(:stage, name: 'Blue stage', script: @course_unit.script, flex_category: Plc::LearningModule::CONTENT_MODULE)
+
+    @stage_honesty = create(:stage, name: 'Honesty stage', script: @course_unit.script, flex_category: Plc::LearningModule::PRACTICE_MODULE)
+    @stage_no_nickname = create(:stage, name: 'No nickname stage', script: @course_unit.script, flex_category: Plc::LearningModule::PRACTICE_MODULE)
+
+    @module_cliffs = create(:plc_learning_module, name: 'Getting thrown off cliffs', plc_course_unit: @course_unit, module_type: Plc::LearningModule::CONTENT_MODULE, stage: @stage_cliffs)
+    @module_ornithology = create(:plc_learning_module, name: 'Ornithology', plc_course_unit: @course_unit, module_type: Plc::LearningModule::CONTENT_MODULE, stage: @stage_ornithology)
+    @module_ignorance = create(:plc_learning_module, name: 'Admitting Ignorance', plc_course_unit: @course_unit, module_type: Plc::LearningModule::CONTENT_MODULE, stage: @stage_ignorance)
+    @module_blue = create(:plc_learning_module, name: 'Blue', plc_course_unit: @course_unit, module_type: Plc::LearningModule::CONTENT_MODULE, stage: @stage_blue)
+
+    @module_honesty = create(:plc_learning_module, name: 'Answering questions honestly', plc_course_unit: @course_unit, module_type: Plc::LearningModule::PRACTICE_MODULE, stage: @stage_honesty)
+    @module_no_nickname = create(:plc_learning_module, name: 'Not revealing your nickname', plc_course_unit: @course_unit, module_type: Plc::LearningModule::PRACTICE_MODULE, stage: @stage_no_nickname)
+
+    q1_dsl = <<DSL
```

Comment:
`DSL.strip_heredoc.chomp` will let you preserve indentation on the following lines.